projects/plugins/jetpack/extensions/blocks/premium-content/buttons/index.js

```diff
@@ -29,6 +30,18 @@ const settings = {
 	edit,
 	save,
 	usesContext: [ 'premium-content/planId', 'premium-content/isPreview' ],
+	deprecated: [
+		{
+			attributes: {},
```

Comment:
I think to get this deprecation working, it also needs to specify the block supports that it's using (if I'm reading the [docs](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/) correctly 🤔). Adding the following works for me:

```suggestion
			attributes: {},
			supports: {
				align: true,
				alignWide: false,
				lightBlockWrapper: true,
				inserter: false,
			},			
```

Then in the console, it looks like the markup is getting updated correctly:

![image](https://user-images.githubusercontent.com/14988353/118204562-d4ed7c00-b4a1-11eb-9f01-660b7ea275d3.png)

Reply:
Thanks!